### ISSUE TITLE
Error building project after upgrading to 0.2.0

### ISSUE THREAD

**Issue by quesodev** [sanity-io/sanity-plugin-asset-source-unsplash#11]
I've reproduced the following error after the following setp:

- Start a new project with "sanity init" and select the movie template
- Run "sanity install sanity-plugin-asset-source-unsplash"
- Add the following to sanity.json parts array:

    {
      "implements": "part:@sanity/form-builder/input/image/asset-sources",
      "path": "./parts/assetSources.js"
    }

- Add the following to ./parts/assetSources.js:

import Unsplash from 'part:sanity-plugin-asset-source-unsplash/image-asset-source'
export default [Unsplash]

RESULT:

webpack building...
⠙ Compiling...webpack built 4b16de742c34a9621d44 in 625ms
✔ Compiling...
Failed to compile.

Error in ./node_modules/react-photo-album/dist/index.esm.js
Module parse failed: Unexpected token (211:37)
You may need an appropriate loader to handle this file type.
| };
| const defaultRenderPhoto = ({ imageProps }) => {
|     const { src, alt, srcSet, sizes, ...rest } = imageProps;
|     return jsx("img", { src: src, alt: alt, ...(srcSet ? { srcSet, sizes } : null), ...rest });
| };
 @ ./node_modules/sanity-plugin-asset-source-unsplash/dist/lib/components/UnsplashAssetSource.js 27:42-70
 @ ./node_modules/sanity-plugin-asset-source-unsplash/dist/lib/index.js
 @ ./node_modules/sanity-plugin-asset-source-unsplash/dist/lib (all:part:@sanity/form-builder/input/image/asset-source)
 @ ./node_modules/@sanity/form-builder/lib/legacyParts.js
 @ ./node_modules/@sanity/form-builder/lib/sanity/formBuilderValueStore.js
 @ ./node_modules/@sanity/form-builder/lib/sanity/legacyPartImplementations/form-builder.js (part:@sanity/form-builder)
 @ ./node_modules/@sanity/desk-tool/lib/panes/document/documentPanel/documentViews/FormView.js
 @ ./node_modules/@sanity/desk-tool/lib/panes/document/documentPanel/documentViews/index.js
 @ ./node_modules/@sanity/desk-tool/lib/panes/document/documentPanel/DocumentPanel.js
 @ ./node_modules/@sanity/desk-tool/lib/panes/document/documentPanel/index.js
 @ ./node_modules/@sanity/desk-tool/lib/panes/document/DocumentPane.js
 @ ./node_modules/@sanity/desk-tool/lib/panes/document/index.js
 @ ./node_modules/@sanity/desk-tool/lib/panes/DeskToolPane.js
 @ ./node_modules/@sanity/desk-tool/lib/panes/index.js
 @ ./node_modules/@sanity/desk-tool/lib/DeskTool.js
 @ ./node_modules/@sanity/desk-tool/lib/_parts/base-tool.js
 @ ./node_modules/@sanity/vision/lib/VisionTool.js (all:part:@sanity/base/tool)
 @ ./node_modules/@sanity/default-layout/lib/util/getRegisteredTools.js
 @ ./node_modules/@sanity/default-layout/lib/router.js
 @ ./node_modules/@sanity/default-layout/lib/Root.js (part:@sanity/base/root)
 @ ./node_modules/@sanity/base/lib/components/SanityRoot.js (part:@sanity/base/sanity-root)
 @ ./node_modules/@sanity/server/lib/browser/entry-dev.js
 @ multi ./node_modules/@sanity/server/lib/browser/entry-dev.js

**Comment by dhstack** [comment]
I'm also running into this issue. Happy to provide more info if needed!

**Comment by tomaszcupial** [comment]
I solved the problem by installing older version of `react-photo-album`. Just run `npm install react-photo-album@1.10.2` in your sanity studio folder and it should work just fine. But this is only a temporary solution.

**Comment by AleaQ** [comment]
in case anyone else is having issues with this, installing the older version of react-photo-album did not work for me.

**Comment by igordanchenko** [comment]
@AleaQ, the correct workaround to pin `react-photo-album` version would be via yarn resolution in your package.json:

```
"resolutiuons" : {
    "react-photo-album": "1.10.2"
}
```

and then running 

```
yarn
yarn start
```

**Comment by igordanchenko** [comment]
TL;DR; Based on what I can see so far, the issue at hand here is that the 4-years old webpack v3.12.0 is unable to consume react-photo-album's ESM bundle. react-photo-album dropped support for UMD bundle in version 1.12.0, so the last version providing UMD bundle is 1.11.1. I'll go ahead and submit a PR to hard-pin react-photo-album's version in sanity-plugin-asset-source-unsplash package.json but that would require a new plugin release.

**Comment by snorrees** [comment]
Fixed in [v0.2.1](https://github.com/sanity-io/sanity-plugin-asset-source-unsplash/releases/tag/v0.2.1) 🙇‍♂️
